debian/autobake-deb.sh

```diff
@@ -155,6 +163,10 @@ in
     exit 1
 esac
 
+if [ ! -f /usr/share/debhelper/dh_package_notes/package-notes.mk ]; then
+  remove_package_notes
```

Comment:
I don't think it's a good idea. This script builds **release packages**. They should be consistently built the way we want them to and not depend on whatever we forgot to install on a particular new builder. Better let it fail here — we'll see it and fix the builder — instead of us suddenly releasing the package that doesn't has what we want

Reply:
Ok. Sure.